pkg/cable/driver.go

```diff
@@ -1,9 +1,17 @@
-package ipsec
+package cable
 
 import (
+	"fmt"
+
+	"github.com/submariner-io/submariner/pkg/cable/ipsec"
 	"github.com/submariner-io/submariner/pkg/types"
 )
 
+const (
+	IPSec     = "ipsec"
```

Comment:
Could you rename this to StrongSwan? We want to add a LibreSwan cable too (see #101 and #107).

Reply:
Done